src/test/java/hudson/plugins/cobertura/CoberturaPublisherTest.java

```diff
@@ -1,49 +0,0 @@
-package hudson.plugins.cobertura;
```

Comment:
Given that the constructor tests were only testing that certain arguments were being properly set, and the change in constructor turned these into trivial setter/getter, the test didn't make sense any more. I could have added all the tests for every single setter/getter, but given that the existing constructor unit tests were not comprehensive (it only tested 5 of the 10 arguments), it didn't seem worth it at the time to rewrite and add all those tests. It could be added if we really want the explicit code coverage for the most trivial setters/getters.